src/CookieJar.php

```diff
@@ -310,7 +310,7 @@ public function merge($arg)
 
 		}, func_get_args());
 
-		$data = call_user_func_array('Arr::merge', $arguments);
+		$data = call_user_func_array(__NAMESPACE__.'\Arr::merge', $arguments);
```

Comment:
I would say alias the Arr class in the tests to the root namespace. That will make the tests run. I will finish the DIC very soon which will make it possible to inject an instance in a setter.